Your Warp script gives Errors during installation.

Hi and thanks for the nice work.

fscarmen's Warp repo has been disabled by github. 

https://github.com/fscarmen/warp

Since your script pulls data from that repository, it chokes in the middle of the installation.  Although fscarmen claims to be back in two days, (I'm assuming he/she is trying to remove some codes to comply with github policies) I hope you have a fork or clone of that repository somewhere in case the main Repo never comes back. 

I also hope there's no automatic update set up anywhere in your script or in 3x-ui because that will most likely break everyone's installation.

Cheers.

same problem . 
'  [ERROR] Sorry, the installation of WARP socks5 proxy (WireProxy) failed! Please try again later.'
as the OP says its due to a problem resulting in the og repo getting suspended

--- a/install_warp_proxy.sh
+++ b/install_warp_proxy.sh
@@ -476,7 +476,7 @@ function step_install_pkgs() {
 function step_create_command() {
   {
     mkdir -p /etc/wireguard
-    wget -N -P /etc/wireguard https://raw.githubusercontent.com/fscarmen/warp/main/menu.sh
+    wget -N -P /etc/wireguard https://gitlab.com/fscarmen/warp/-/raw/main/menu.sh
     chmod +x /etc/wireguard/menu.sh
     ln -sf /etc/wireguard/menu.sh /usr/bin/warp
   }

